lib/ext/fhir/library.rb

```diff
@@ -0,0 +1,23 @@
+# Custom functionalty for fhir-mongoid-models/app/models/fhir/library.rb
+module FHIR
+  class Library < DomainResource
+
+    def create_data_elements(fhir_value_sets)
```

Comment:
Codacy found an issue: [Assignment Branch Condition size for create_data_elements is too high. [<5, 17, 1> 17.75/15]](https://app.codacy.com/gh/MeasureAuthoringTool/fhir-mongoid-models/pullRequest?prid=5999190)